### DISCUSSION TITLE
Postgrebase officially starts!

### DISCUSSION THREAD

**Discussion by zhenruyan** [zhenruyan/postgrebase#1] [Announcements]
Experienced confusion, search, and exploration. Postgrebase can finally run normally.

But this is not enough, our goal is to make it able to withstand high levels of online pressure and support mainstream databases such as PostgreSQL.

Cache synchronization and minio based file synchronization

To achieve multi node operation

**Comment by c-nv-s** [comment]
Looking forward to seeing if you can keep this project updated.
There is also another similar project https://github.com/staticbackendhq/core

**Reply by zhenruyan** [reply]
Due to the good and stable performance of PG and GO, there are currently no major updates

**Reply by zhenruyan** [reply]
However, there are plans to add annotation and collection functions for tables, as well as a few bug fixes

**Reply by c-nv-s** [reply]
maybe compatibility with the v0.23 branch of pocketbase should be a goal, it has dropped the Echo router for new routing features in the std library... maybe

**Reply by zhenruyan** [reply]
> maybe compatibility with the v0.23 branch of pocketbase should be a goal, it has dropped the Echo router for new routing features in the std library... 
Is there anything wrong with the routing of echo? Although I prefer bunrouter
My goal is to improve database compatibility and business usability

**Reply by c-nv-s** [reply]
no nothing wrong, I'm just mentioning that you should check the changes from the new branch going forward if you want to maintain some kind of alignment that's all. The router changes are not the only thing that has changed.
